src/classes/GE_TemplateBuilderCtrl.cls

```diff
@@ -47,6 +47,13 @@ public with sharing class GE_TemplateBuilderCtrl {
     */
     private static final String TEMPLATE_VERSION = '1.0';
 
+    /*******************************************************************************************************
+    * @description List of required DataImportBatch__c fields
+    */
+    private static final String[] REQUIRED_BATCH_HEADER_FIELDS = new String[]{
```

Comment:
Yeah, I think we've still got some open questions around this. I created this variable solely because the WI called for requiring the Batch Name field, but the field on the actual batch object isn't required. I think we have respecting the required-ness of a field based on the actual field describe as a later work item. But yeah it's scenario in where we have the concept of field required-ness is multiple places. It'd be nice to have that in one place and ideally from the field describe.

Reply:
I see!  Ok, since we are making it required for the lwc UI then, lets house this in the lwc controller instead of the Apex side.

Reply:
All set! This has been moved into the web component.